openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -152,7 +152,11 @@ def process(self, instance):
                         json.dumps(preset, indent=4, sort_keys=True)
                     )
                 )
-
+            if (
+                preset["viewport_options"].get("reloadTextures")
+                and "textures" in preset["viewport_options"]
```

Comment:
Looking at the logic [here](https://github.com/ynput/OpenPype/pull/5942/files#diff-34078a2a4bf0a4e1e51eda1d039f05d7557621c0a8346706b30967647c9d12b8R175) the `"textures" in preset["viewport_options"]` is incorrect. The other code seems to assume that the key does exist in viewport options, it does:

```python
if preset["viewport_options"][key]:
``` 

So it's likely always there?


Which makes me think you might have intended to check here whether it's a true value or not and thus should just be:
```suggestion
                and preset["viewport_options"]["textures"]
```
Or if you're still unsure whether it exists:

```suggestion
                and preset["viewport_options"].get("textures")
```
But as stated the Extract Playblast will still always try to access the key anyway.

Reply:
it only checks to see if textures and reload textures options in the setting being enabled at the same time, If Yes, it will reload textures.